tests/integration/backward_compatible/listener_test.py

```diff
@@ -6,80 +8,77 @@
     wait_for_partition_table,
 )
 
+LISTENER_TYPES = [
+    (
+        "smart",
+        True,
+    ),
+    (
+        "non-smart",
+        False,
+    ),
+]
+
 
-class ListenerTest(HazelcastTestCase):
+class ListenerRemoveMemberTest(HazelcastTestCase):
     def setUp(self):
         self.rc = self.create_rc()
         self.cluster = self.create_cluster(self.rc, None)
         self.m1 = self.cluster.start_member()
         self.m2 = self.cluster.start_member()
         self.client_config = {
             "cluster_name": self.cluster.id,
+            "heartbeat_interval": 1.0,
```

Comment:
Since the member is terminated forcefully (see tests/base.py/_Member#shutdown), we detect connection loss through heartbeat pings. So, the smaller the interval, the faster our tests will be